sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/HDFSBackedStateStoreProvider.scala

```diff
@@ -201,6 +201,11 @@ private[sql] class HDFSBackedStateStoreProvider extends StateStoreProvider with
     override def toString(): String = {
       s"HDFSStateStore[id=(op=${id.operatorId},part=${id.partitionId}),dir=$baseDir]"
     }
+
+    override def removeColFamilyIfExists(colFamilyName: String): Unit = {
+      throw new UnsupportedOperationException("Removing with " +
```

Comment:
nit: Do you plan to add this to error class framework in another PR? Regardless, maybe better to clarify removing with what. Let's not pretend they can get context about error from stack trace.

Reply:
Yeah, I wanted to get the NERF PR merged, first, will rebase this PR after that one is merged to keep this change isolated.

Reply:
FYI I just merged error class PR.

Reply:
Just rebased and added the error classes